src/web/app/pages-instructor/instructor-session-edit-page/copy-questions-from-other-sessions-modal/copy-questions-from-other-sessions-modal.component.html

```diff
@@ -5,54 +5,74 @@ <h4 class="modal-title">Copy Questions</h4>
+      <button id="sort-creation-date" class="btn btn-light" [disabled]="isSelectedForSorting(SortBy.SESSION_CREATION_DATE)" (click)="sortFeedbackSessionTabs(SortBy.SESSION_CREATION_DATE)">Creation Date</button>
+    </div>
+  </div>
+  <div class="feedback-session-tabs" *ngFor="let feedbackSessionTabModel of feedbackSessionTabModels">
+    <div class="card">
+      <div class="card-header cursor-pointer bg-primary text-white" (click)="toggleCard(feedbackSessionTabModel)">
+        <strong class="text-break">[{{feedbackSessionTabModel.courseId}}] Session: {{feedbackSessionTabModel.feedbackSessionName}}</strong>
+        <div class="card-header-btn-toolbar">
+          <tm-panel-chevron [isExpanded]="feedbackSessionTabModel.isTabExpanded"></tm-panel-chevron>
+        </div>
+      </div>
+
+      <div *ngIf="feedbackSessionTabModel.isTabExpanded">
+        <tm-loading-retry [shouldShowRetry]="feedbackSessionTabModel.hasLoadingFailed" [message]="'Failed to load questions'" (retryEvent)="loadQuestions(feedbackSessionTabModel)">
+          <div *tmIsLoading="!feedbackSessionTabModel.hasQuestionsLoaded">
+            <div *ngIf="!feedbackSessionTabModel.hasLoadingFailed" class="card-body">
+              <ng-container *ngIf="feedbackSessionTabModel.hasQuestionsLoaded && feedbackSessionTabModel.questionsTableRowModels.length > 0, else noQuestionsTemplate">
+                <table id="copy-question-table" class="table table-bordered table-responsive-lg">
+                  <thead class="bg-primary text-white">
+                    <tr>
+                      <th>#</th>
+                      <th class="sortable-header" (click)="sortQuestionsToCopyForFeedbackSession(feedbackSessionTabModel, SortBy.QUESTION_TYPE)">
+                        Question Type
+                        <span class="fa-stack">
+                          <i class="fas fa-sort"></i>
+                          <i *ngIf="feedbackSessionTabModel.questionsTableRowModelsSortBy === SortBy.QUESTION_TYPE && feedbackSessionTabModel.questionsTableRowModelsSortOrder === SortOrder.DESC" class="fas fa-sort-down"></i>
+                          <i *ngIf="feedbackSessionTabModel.questionsTableRowModelsSortBy === SortBy.QUESTION_TYPE && feedbackSessionTabModel.questionsTableRowModelsSortOrder === SortOrder.ASC" class="fas fa-sort-up"></i>
+                        </span>
+                      </th>
+                      <th class="sortable-header" (click)="sortQuestionsToCopyForFeedbackSession(feedbackSessionTabModel, SortBy.QUESTION_TEXT)">
+                        Question Text
+                        <span class="fa-stack">
+                          <i class="fas fa-sort"></i>
+                          <i *ngIf="feedbackSessionTabModel.questionsTableRowModelsSortBy === SortBy.QUESTION_TEXT && feedbackSessionTabModel.questionsTableRowModelsSortOrder === SortOrder.DESC" class="fas fa-sort-down"></i>
```

Comment:
I feel like we can shorten these 2 lines by putting these 2 conditions into a function in the ts file, so smth like this
```
isChevronUp() {
 if (feedbackSessionTabModel.questionsTableRowModelsSortBy === SortBy.QUESTION_TEXT && feedbackSessionTabModel.questionsTableRowModelsSortOrder === SortOrder.DESC) return true;
 return false;
}
```

Reply:
Thanks for the suggestion! I have created a helper function `isSortQuestionsBy` for this.